sample-client-app/sample-app-with-shared-lib/README.md

```diff
@@ -0,0 +1,36 @@
+Singleton Java Client Sample Application with Shared Library
+============
+
+This application can be used to demonstrate usage of a shared library that uses the Singleton Java client.
```

Comment:
demonstrate usage of a shared library 
may should be:
demonstrate usage of an application using a shared library

Reply:
No, what I put is what I actually meant.. to. demonstrate usage of (how to use) a shared library that uses the Singleton Java client